app/src/main/java/com/eventyay/organizer/data/faq/FaqDao.java

```diff
@@ -0,0 +1,23 @@
+package com.eventyay.organizer.data.faq;
+
+import androidx.room.Dao;
+import androidx.room.Insert;
+import androidx.room.OnConflictStrategy;
+import androidx.room.Query;
+
+import java.util.List;
+
+import io.reactivex.Observable;
+
+@Dao
+public interface FaqDao {
+
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    void insertFaq(Faq faq);
```

Comment:
Same

Reply:
Shouldn't we use `Observable<Faq>` here?

Reply:
Updated and working.